test/membrane_live/webinars_test.exs

```diff
@@ -33,15 +34,17 @@ defmodule MembraneLive.WebinarsTest do
         "title" => "some title"
       }
 
-      assert {:ok, %Webinar{} = webinar} = Webinars.create_webinar(valid_attrs)
+      user = user_fixture()
```

Comment:
As we call this `user_fixture` multiple time we could move this to constant or setup.